src/helpers/consul/index.js

```diff
@@ -15,7 +15,7 @@ export const getConsulAuthToken = async () => await SecureStore.getItemAsync(CON
 
 export const setConsulUser = async (user) => {
   if (user) {
-    await SecureStore.setItemAsync(CONSUL_USER, JSON.stringify(user));
+    await SecureStore.setItemAsync(CONSUL_USER, user.id);
```

Comment:
could we pass the `userId` directly to this function instead of the whole `user`? this way we always know to receive a number instead of some object with values we cannot know 100%

Reply:
Yes, that's right. We only need the id information anyway.


https://github.com/ikuseiGmbH/smart-village-app-app/pull/589/commits/75e5cac47ab9ecaa53e670a89cb4c2bbd96025be